kythe/cxx/indexer/cxx/ImputedConstructorSupport.h

```diff
@@ -32,9 +32,9 @@ namespace kythe {
 class ImputedConstructorSupport : public LibrarySupport {
  public:
   explicit ImputedConstructorSupport(
-      std::unordered_set<std::string> allowed_constructor_patterns = {
```

Comment:
Sorry, I meant that it should be "allowed_constructor_patterns" rather than "allowed_constructor_names" (because they are regular expressions)